sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -59,25 +59,23 @@
  *
  * pipeline.apply(JdbcIO.read()
  *   .withDataSource(myDataSource)
- *   .withRowMapper(new JdbcIO.RowMapper<MyElement>() {
- *     public MyElement mapRow(ResultSet resultSet) {
- *       // use the resultSet to build the element of the PCollection
- *       // for instance:
- *       // return resultSet.getString(2);
+ *   .withStatement("select id,name from Person")
```

Comment:
For the read, it should be query. I'm using common terminology: query is something read-only; statement is something that modifies things.